core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunner.java

```diff
@@ -595,6 +596,11 @@ private void decoratePillarWithTags(Stack stack, Map<String, SaltPillarPropertie
         }
     }
 
+    private void saveUnboundEliminationPillar(Map<String, SaltPillarProperties> servicePillar, String accountId) {
+        servicePillar.put("elimination", new SaltPillarProperties("/unbound/elimination.sls", singletonMap("supported",
+                entitlementService.isUnboundEliminationSupported(accountId))));
+    }
```

Comment:
could you return with a map instead of passing `servicePillar` as an argument? it's more clear
see: `servicePillar.putAll(saveSssdIpaPillar(kerberosConfig, serviceLocations, stack.getEnvironmentCrn()));`